docs/configration.md

```diff
@@ -0,0 +1,111 @@
+# Protect.js Configuration
+
+Protect.js is configured with [toml](https://toml.io/en/) files or environment variables, and is used to initialize the client when Protect.js is initialized.
+
+Environment variables will take precedence over configuration files and it's recommented to use them for sensitive values.
+
+## Jump to a section
+
+- [cipherstash.toml](#cipherstashtoml)
+- [cipherstash.secret.toml](#cipherstashsecrettoml)
+- [Environment variables](#environment-variables)
+- [Deploying to production](#deploying-to-production)
+
+## cipherstash.toml
+
+The `cipherstash.toml` file is the primary (non-sensitive) configuration file.
+It stores essential identifiers, like `client_id` and `workspace_id`, but **must not** contain **sensitive secrets** (e.g., `client_key`, `access_key`).
+Those sensitive values should be placed in either: 
+
+- The `cipherstash.secret.toml` file, or
+- Environment variables (e.g., `CS_CLIENT_KEY`).
+
+The `cipherstash.toml` must be stored in the root directory of your project.
+
+### Specification
+
+The minimal `cipherstash.toml` file:
+
+```toml
+# cipherstash.toml
+
+[encrypt]
+client_id = "your-client-id"
+
+[auth]
+workspace_id = "your-workspace-id"
+```
+
+#### `[encrypt]` section
+
+- `client_id` (**required**): A UUID that identifies the client or project.  
+- `default_keyset_id` (optional): A UUID referencing the default keyset (used if you have more than one keyset).  
+- `client_key` (**not allowed**): This is explicitly disallowed and will be rejected at runtime.
+The client key must be stored in `cipherstash.secret.toml` or in an environment variable `CS_CLIENT_KEY`.
+
+#### `[auth]` section
+
+- `workspace_id` (**required**): A base-32 encoded encodings of 10-byte identifier that uniquely identifies the workspace.
```

Comment:
Remove the extra "encodings of". "A base-32 encoded 10-byte identifier" is fine.